### DISCUSSION TITLE
Add notification on acmebot

### DISCUSSION THREAD

**Discussion by ash06dec** [polymind-inc/acmebot#526] [Q&A] [answered]
Is it possible to add a notification to the email address while doing certificate :
- Creation
- Renew
- Deletion

**Comment by shibayan** [comment, marked as answer]
Email is not planned to be supported due to problems with the sending server. Instead, you can use Webhooks to send to Slack / Teams, etc.

**Reply by ash06dec** [reply]
can you please elaborate or send me some blogs or document.

**Reply by shibayan** [reply]
Documentation exists for Webhook.
https://github.com/shibayan/keyvault-acmebot/wiki/Getting-Started#2-add-application-settings

**Reply by ash06dec** [reply]
unfortunately incoming webhook is disabled in the teams channel at my organization, any other way ?

**Reply by shibayan** [reply]
Acmebot supports sending to any webhook, so it would be possible to create a workflow that would receive it in Logic Apps and send it via email, etc.

Beyond this point, the world is not related to Acmebot.
